IMAGES.rst

```diff
@@ -230,45 +229,41 @@ or
 Naming conventions
 ==================
 
-By default images are pulled and pushed from and to Github Container registry when you use Breeze's push-image
-or build commands.
-
-We are using GitHub Container Registry as build cache.The images are all in organization wide "apache/"
-namespace. We are adding "airflow-" as prefix for image names of all Airflow images.
-The images are linked to the repository via ``org.opencontainers.image.source`` label in the image.
+By default images we are using cache for images in Github Container registry. We are using GitHub
+Container Registry as development image cache and CI registry for build images.
+The images are all in organization wide "apache/" namespace. We are adding "airflow-" as prefix for
+the image names of all Airflow images. The images are linked to the repository
+via ``org.opencontainers.image.source`` label in the image.
 
 See https://docs.github.com/en/packages/learn-github-packages/connecting-a-repository-to-a-package
 
 Naming convention for the GitHub packages.
 
-Images with a commit SHA (built for pull requests and pushes)
+Images with a commit SHA (built for pull requests and pushes). Those are images that are snapshot of the
+currently run build. They are built once per each build and pulled by each test job.
 
 .. code-block:: bash
 
   ghcr.io/apache/airflow/<BRANCH>/ci/python<X.Y>:<COMMIT_SHA>         - for CI images
   ghcr.io/apache/airflow/<BRANCH>/prod/python<X.Y>:<COMMIT_SHA>       - for production images
 
-We do not push Base Python images and prod-build images when we prepare COMMIT builds, because those
-images are never rebuilt locally, so there is no need to store base images specific for those builds.
 
-Latest images (pushed when main merge succeeds):
+The cache images (pushed when main merge succeeds) are kept with ``cache`` tag:
 
 .. code-block:: bash
 
-  ghcr.io/apache/airflow/<BRANCH>/python:<X.Y>-slim-buster        - for base Python images
-  ghcr.io/apache/airflow/<BRANCH>/ci/python<X.Y>:latest           - for CI images
-  ghcr.io/apache/airflow/<BRANCH>/ci-manifest/python<X.Y>:latest  - for CI Manifest images
-  ghcr.io/apache/airflow/<BRANCH>/prod/python<X.Y>:latest         - for production images
-  ghcr.io/apache/airflow/<BRANCH>/prod-build/python<X.Y>:latest   - for production build stage
```

Comment:
* 3/5 of the images can be scraped now. Just :cache for PROD and CI are enough to efficiently rebuild the image.